docs/source/05-03-when-youve-installed-talk.md

```diff
@@ -44,13 +44,13 @@ Your plans might include:
 
 ![[IMAGE] A screenshot of the pre-moderate option in the Configure tab](http://blog.coralproject.net/wp-content/uploads/2018/03/premod.png)
```

Comment:
Missed an image here.

Reply:
I don't think so - the blog images are redirecting fine, for some reason it's only the articles that don't. Does the image not show up for you?

Reply:
It does redirect fine, I just expect that we could swap out the old url for the new one, less redirects means faster loads :)